internal/templates/testing.tmpl

```diff
@@ -26,27 +26,17 @@ type Testing{{.BackendsName}} struct {
 {{end}}{{end}}
 }
 
-{{range .Deps}}
+{{range .Deps}}{{if not .IsDuplicate }}
```

Comment:
where is IsDuplicate defined? is that new? i don't get how it relates to the filter deps change